packages/flame/lib/src/camera/behaviors/follow_behavior.dart

```diff
@@ -59,19 +62,17 @@ class FollowBehavior extends Component {
 
   @override
   void update(double dt) {
-    final delta = target.position - owner.position;
-    if (horizontalOnly) {
-      delta.y = 0;
-    }
-    if (verticalOnly) {
-      delta.x = 0;
-    }
-    final distance = delta.length;
+    _tempDelta.setValues(
+      verticalOnly ? 0 : target.position.x - owner.position.x,
+      horizontalOnly ? 0 : target.position.y - owner.position.y,
+    );
+
+    final distance = _tempDelta.length;
     if (distance > _speed * dt) {
-      delta.scale(_speed * dt / distance);
+      _tempDelta.scale(_speed * dt / distance);
```

Comment:
nit: if you want you can extract `_speed * dt` to avoid computing it twice

Reply:
Done.